src/Sulu/Bundle/ContentBundle/Search/Metadata/StructureProvider.php

```diff
@@ -189,6 +188,12 @@ public function loadMetadataForClass(\ReflectionClass $class)
             ));
         }
 
+        $indexMeta->addFieldMapping(self::FIELD_STRUCTURE_TYPE, array(
+            'type' => 'string',
+            'index_strategy' => Field::INDEX_STORED_INDEXED,
```

Comment:
This should be STORED, NOT INDEXED. Wait for https://github.com/massiveart/MassiveSearchBundle/pull/44 to be merged.